pyonfx/utils.py

```diff
@@ -16,14 +16,15 @@
 # along with this program. If not, see http://www.gnu.org/licenses/.
 
 from __future__ import annotations
+
 import math
 import re
-from typing import List, Union, TYPE_CHECKING
+from typing import List, TypeVar, Union
 
-from .convert import Convert, ColorModel
+from .ass_core import Char, Line, Syllable, Word
```

Comment:
Removing TYPE_CHECKING does work fine? It should return an error due to cyclic imports. Also (minor), import orders is not the usual.

Reply:
tests pass and it works in an example file